### DISCUSSION TITLE
Rows keep changing to 0 value

### DISCUSSION THREAD

**Discussion by organicnz** [AntlerVC/firetable#802] [Support / Q&A] [answered]
Hi folks,

How to fix this bug with with rows? They keep changing to 0 value when I'm importing from Google spreadsheet for some reason :)

Thank you :)

<img width="1440" alt="Screen Shot 2022-08-30 at 6 24 34 PM" src="https://user-images.githubusercontent.com/4765090/187477545-ee048c18-ff96-46bb-8a8a-1227db4f5102.png">

**Comment by harinij** [comment, marked as answer]
Hi @organicnz it is not super clear what issue you are facing. Would it be possible for you to share a video recording to show the issue you are facing?

We have a detailed video + blog here on how to import data using CSV: https://www.rowy.io/blog/import-csv-to-firestore
